arcgis-ios-sdk-samples/Maps/Create and save a map/SaveAsViewController.swift

```diff
@@ -64,7 +64,7 @@ class SaveAsViewController: UITableViewController {
         // The titles of the portal folders and a "None" option.
         let folderTitles = portalFolders.map { $0.title ?? "(No Title)" } + ["None"]
```

Comment:
Because empty selection is now supported, there is no need for the `None` option anymore. Could you please remove it?